.appveyor.yml

```diff
@@ -1,15 +1,23 @@
 platform:
 - x64
+
 environment:
-  nodejs_version: "6"
```

Comment:
As much as I like being thorough and testing all LTS & current node releases, this does blow up the test time. Since we are doing all 3 on Travis (in parallel FTW) we could probably go back to just testing minimum supported node version on AppVeyor.